### ISSUE TITLE
maintitle, labeltitle and extratitle

### ISSUE THREAD

**Issue by maieul** [plk/biblatex#421]
Considering the MWE

```
\documentclass{article}
\usepackage[labeltitle=true]{biblatex}
\usepackage{filecontents}

\begin{filecontents}{\jobname.bib}



@mvbook{Ehrhard,
	Address = {Osnabrück},
	Author = {Albert Ehrhard},
	Origdate = {1937/1952},
	Origlocation = {Leipzig},
	Origpublisher = {J. C. Hinrichs Verlag},
	Publisher = {Otto Zeller Verlag},
	Subtitle = {von den Anfängen bis zum Ende des 16. Jahrhunderts},
	Title = {Überlieferung und Bestand der hagiographischen und homiletischen Literatur der griechischen Kirche},
	Volumes = {3},
	Year = {1965}}

@book{Ehrhard1.1,
	Crossref = {Ehrhard},
	Origdate = {1937},
	Part = {1},
	Volume = {1}
}

@book{Ehrhard1.2,
	Crossref = {Ehrhard},
	Origdate = {1938},
	Part = {2},
	Volume = {1},
}

@book{Ehrhard1.3,
	Crossref = {Ehrhard},
	Date-Modified = {2014-06-03 12:48:11 +0000},
	Keywords = {a verifier,prague_secondaire},
	Origdate = {1939/1952},
	Origlocation = {Leipzig},
	Origpublisher = {J. C. Hinrichs Verlag},
	Part = {3},
	Publisher = {Otto Zeller Verlag},
	Volume = {1},
	Year = {1965}
}
\end{filecontents}
\bibliography{\jobname.bib}
\begin{document}
\nocite{*}
s
\end{document}
```

As the subentry have not individual title, it is not possible to distinguish them in abreviated form. As they have no label title, they have no extratitle.

Should not be relevant to move default declaration of labeltitle to 

````
\DeclareLabeltitle{%
  \field{shorttitle}
  \field{title}
  \field{maintitle}
}
````
?

**Comment by plk** [comment]
I can see your point but I'm not sure this should be the default since if you are using such crossrefs (a lot of people are not ...), you can just change the definition ....

**Comment by maieul** [comment]
As I don't see any problem for people which do not use it, it seems me should be better for other people �

**Comment by plk** [comment]
Well, ok, that's fair enough but since I released biblatex 3.4 a couple of days ago, it will have to wait until 3.5, sorry ....

**Comment by maieul** [comment]
not hurry !
release of biblatex 3.4 is the more important
